MYshort.bib

```diff
@@ -518,9 +547,83 @@ @String{VLDB
 @String{Wiley = "John Wiley \& Sons"}
  	@String{WileyMonterey = "Monterey, CA, USA"} % This is the address of the responsible editors
 
-other
+@String{Sydney = "Sydney, Australia"}
+@String{WaterlooAustralia = "Waterloo, Australia"}
+
+@String{Ontario = "Ontario, Canada"}
+
+@String{KongensLyngby = "Kongens Lyngby, Denmark"}
+@String{Copenhagen = "Copenhagen, Denmark"}
+
+@String{Aachen = "Aachen, Germany"}
+@String{Essen = "Essen, Germany"}
+@String{Cologne = "Cologne, Germany"}
+@String{Karlsruhe = "Karlsruhe, Germany"}
+@String{Maastricht = "D{\"u}ren/Maastricht, Germany"}
+@String{Nuremberg = "Nuremberg, Germany"}
+@String{Paderborn = "Paderborn, Germany"}
+
+@String{Lyon = "Lyon, France"}
+
+@String{Limerick = "Limerick, Ireland"}
+
+@String{Mahebourg = "Mahebourg, Republic of Mauritius"}
+
+@String{Amsterdam = "Amsterdam, Netherlands"}
+
+@String{Gothenburg = "Gothenburg, Sweden"}
+@String{Vasteras = "V{\"a}ster\r{a}s, Sweden"}
+@String{Stockholm = "Stockholm, Sweden"}
+
+@String{Geneva = "Geneva, Switzerland"}
+
+@String{Chichester = "Chichester, UK"}
+@String{Hertfordshire = "Hertfordshire, UK"}
+@String{Lancaster = "Lancaster, UK"}
+@String{Oxford = "Oxford, UK"}
+@String{Stevenage = "Stevenage, UK"}
+
+@String{Kyiv = "Kyiv, Ukraine"}
+
+@String{Ames = "Ames, IA, USA"}
+@String{Berkeley = "Berkeley, CA, USA"}
+@String{BocaRaton = "Boca Raton, FL, USA"}
+@String{Boston = "Boston, MA, USA"}
+@String{Burlington = "Burlington, MA, USA"}
+@String{Duluth = "Duluth, MN, USA"}
+@String{ElPaso = "El Paso, Texas, USA"}
+@String{ElSegundo = "El Segundo, CA, USA"}
+@String{Hershey = "Hershey, PA, USA"}
+@String{Hingham = "Hingham, MA, USA"}
+@String{Hoboken = "Hoboken, NJ, USA"}
+@String{LaCanadaFlintridge = "La Canada Flintridge, CA, USA"}
+@String{Linthicum = "Linthicum, MD, USA"}
+@String{LosAlamitos = "Los Alamitos, CA, USA"}
+@String{Newark = "Newark, NJ, USA"}
+@String{Newton = "Newton, MA, USA"}
+@String{NewYork = "New York, NY, USA"}
+@String{Norwell = "Norwell, MA, USA"}
+@String{Orlando = "Orlando, FL, USA"}
+@String{RedBank = "Red Bank, NJ, USA"}
+@String{Richland = "Richland, SC, USA"}
+@String{Riverton = "Riverton, NJ, USA"}
+@String{StPaul = "St. Paul, MN, USA"}
+@String{SanBernardino = "San Bernardino, CA, USA"}
+@String{SanDiego = "San Diego, CA, USA"}
+@String{SanFrancisco = "San Francisco, CA, USA"}
+@String{Sebastopol = "Sebastopol, CA, USA"}
+@String{Secaucus = "Secaucus, NJ, USA"}
+@String{WestLafayette = "West Lafayette, Indiana"}
+
+% other
 @String{MIT = "MIT"}
 @String{CoRR = "CoRR"}
 @String{GTTSE = "GTTSE"}
 @String{ReMoDD = "ReMoDD"}
 @String{LNCS = "LNCS"}
+
+% type
+@String{Master = "Master's Thesis"}
+@String{Bachelor = "Bachelor's Thesis"}
+@String{Project = "Project Thesis"}
+@String{PhD = "PhD Thesis"}
```

Comment:
Maybe this PR would be a good time to extract all these duplicate variables for cities and countries from MYabrv and MYshort?

Reply:
We could include strings that are common to both files directly in the literature.bib file.
But we can do this at any time. It does not have to be within this PR.

Reply:
We could. This would also be a somewhat breaking change though and when doing it in this PR, there would be just one instead of two such merges.